tardis/simulation/base.py

```diff
@@ -162,6 +167,16 @@ def __init__(
                 f"- input is {convergence_strategy.type}"
             )
 
+        if cplots_kwargs["show_plots"] is not False:
```

Comment:
```suggestion
        if cplots_kwargs["show_plots"]:
```
or ` if cplots_kwargs["show_plots"] is True:`

Why not keep it simple :smile: